frontend/store/surplus.ts

```diff
@@ -92,6 +98,27 @@ export const mutations = {
     setAuctionState(state: State, { auctionId, value }: { auctionId: number; value: AuctionState }) {
         Vue.set(state.auctionStates, auctionId, value);
     },
+    setIsMarketPriceLoading(state: State, isLoading: boolean) {
```

Comment:
I would remove all of this logic from the store. I think we should have the following pattern in the core when retrieving auctions: 

- fetch all auctions
- enrich each auction with transaction fees
- enrich each auction with market price
- return all auctions enriched with every value we need

This is how we do it with the swap auctions. 
https://github.com/sidestream-tech/unified-auctions-ui/blob/main/core/src/auctions.ts#L143

This will make it a lot easier when we want to interact with surplus auctions in other parts of the project (such as the bot), as we would not need to rebuild the logic in there.